crates/ruff_server/docs/setup/NEOVIM.md

```diff
@@ -0,0 +1,52 @@
+## NeoVim Setup Guide for `ruff server`
```

Comment:
I think it should just be "Neovim" everywhere without the uppercase "V". Refer:
* Website: https://neovim.io/
* Repository: https://github.com/neovim/neovim